invenio_pidstore/__init__.py

```diff
@@ -187,7 +187,17 @@
 Invenio-Records installed in order for this example to work):
 
 >>> from invenio_pidstore.resolver import Resolver
->>> from invenio_records.api import Record
+>>> try:
+...     from invenio_records.api import Record
+... except ImportError:
+...     class Record(dict):
```

Comment:
The primary usage of the resolver is together with ``invenio_records.api.Record``, so I think this is also what the documentation should clearly communicate as it currently does. The example is only an issue because it is being executed, so perhaps it's possible to skip the test of this specific part. I'm unfortunately not in the office today or tomorrow otherwise we could have discussed IRL.